seeds/sudoswap/sudoswap_ethereum_example_trades.csv

```diff
@@ -0,0 +1,24 @@
+block_number,tx_hash,trade_category,nft_contract_address,token_id,amount_original,platform_fee_amount,pool_fee_amount
```

Comment:
when adding a new seed file, the dbt project yaml file should be edited to include the new directory and file for seeds. please see line 148 forward in here: https://github.com/duneanalytics/spellbook/blob/main/dbt_project.yml

Reply:
:heavy_check_mark: